sdk/resources/Azure.ResourceManager.Resources/src/Generated/Models/ApplicationBillingDetails.Serialization.cs

```diff
@@ -10,9 +10,9 @@
 
 namespace Azure.ResourceManager.Resources.Models
 {
-    internal partial class ApplicationBillingDetailsDefinition
+    internal partial class ApplicationBillingDetails
```

Comment:
I think the intent of this comment https://apiview.dev/Assemblies/Review/4f2a3b8e84644efcb90f782e63105dfe#e5235cc9d5e843a2b8ab3d167d4c5059 was to apply to everything that starts with Application to be consistent across all the names